src/test/api-test-utils.ts

```
@@ -0,0 +1,96 @@
+            'eq', 'order', 'limit', 'gt', 'lt', 'gte', 'lte',
+            'in', 'single', 'not', 'neq', 'is', 'or', 'and'
+        ];
+
+        methods.forEach(method => {
+            builder[method] = vi.fn((...args) => builder);
+        });
+
+        // Promise-like behavior: make it awaitable
+        builder.then = vi.fn((resolve, reject) => {
+            // デフォルトでは空の結果を返す
+            return Promise.resolve({ data: null, error: null }).then(resolve, reject);
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)